includes/class-metabox.php

```
@@ -311,21 +311,28 @@ private function save_asset_manager_settings( $post_id ) {
+		 *
+		 * @param int    $post_id       The ID of the post being saved.
+		 * @param string $post_key      The key to look for in $_POST.
+		 * @param array  $valid_handles An array of valid handles to whitelist against.
+		 * @param string $meta_key      The post meta key to save the disabled assets to.
+		 * @since NEXT
+		 */
+		private function process_and_save_assets( $post_id, $post_key, $valid_handles, $meta_key ) {
+			$disabled_assets = array();
+			// phpcs:disable WordPress.Security.NonceVerification.Missing -- Nonce is verified in the calling method `save_asset_manager_settings`.
+			if ( isset( $_POST[ $post_key ] ) && is_array( $_POST[ $post_key ] ) ) {
+				$submitted       = array_map( 'sanitize_text_field', wp_unslash( $_POST[ $post_key ] ) );
```

PHPStan (2.2.2)

[warning] Parameter `#1` $callback of function array_map expects (callable(mixed): mixed)|null, 'sanitize_text_field' given.


(argument.type)